src/modules/zerion-api/requests/wallet-get-meta.ts

```diff
@@ -133,3 +134,22 @@ export function getWalletsMeta(
   const endpoint = `wallet/get-meta/v1?${params}`;
   return ZerionHttpClient.get<Response>({ endpoint, ...options });
 }
+
+function splitIntoChunks<T>(arr: T[], size: number) {
+  const chunks: T[][] = [];
+  for (let i = 0; i < arr.length; i += size) {
+    chunks.push(arr.slice(i, i + size));
+  }
+  return chunks;
+}
+
+export async function getWalletsMetaByChunks(addresses: string[]) {
+  const chunks = splitIntoChunks(addresses, 10);
+  const results = await Promise.all(
+    chunks.map((chunk) => getWalletsMeta({ identifiers: chunk }))
```

Comment:
getWalletsMeta is supposed to be called as a method of `ZerionAPI`
So `getWalletsMetaByChunks` should be defined outside and call it correctly.

But it's not that crucial for how. I'm ok to leave it like this.

Reply:
Or maybe it's even ok to be here. If we need to pass `this` context later we can apply it to `getWalletsMeta` right here

Reply:
I was hesitating to put it here...